IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/ViewControllers/AddPersonViewController.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  AddPersonViewController.swift
+//  SwiftSeedProject
+//
+//  Created by Mugu on 3/22/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import UIKit
+import CoreData
+
+class AddPersonViewController: UIViewController {
+
+    @IBOutlet weak var txtFieldName: UITextField!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    @IBAction func btnAddPersonTouchUpInsided(_ sender: Any) {
+        let context = (UIApplication.shared.delegate as! AppDelegate).persistentContainer.viewContext //Not sure if there's a better way to handle context rather than this way.
```

Comment:
Also, there's the performance hit of `persistentContainer` initialization.. and it's better to hide that on splash/loading screen instead of a user interaction like this button.